readme.md

```diff
@@ -0,0 +1,206 @@
+# unist-lsp
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Size][size-badge]][size]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+Utility to convert between [unist][] and  [language server protocol][].
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unistPointToLspPosition(point)`](#unistpointtolsppositionpoint)
+    *   [`unist-lsp(position)`](#unist-lspposition)
+    *   [`unistPositionToLspRange(position)`](#unistpositiontolsprangeposition)
+    *   [`lspRangeToUnistPosition(range)`](#lsprangetounistpositionrange)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a utility that takes converts between [unist][] and
+[language server protocol][].
+
+## When should I use this?
+
+This project is useful when you want to deal with unist ASTs and a language
+server.
+
+## Install
+
+This package is [ESM only][esm].
+In Node.js (version 14.14+, or 16.0+), install with [npm][]:
+
+```sh
+npm install unist-lsp
+```
+
+In Deno with [`esm.sh`][esmsh]:
+
+```js
+import {
+  lspPositionToUnistPoint,
+  lspRangeToUnistPosition,
+  unistPointToLspPosition,
+  unistPositionToLspRange,
+} from "https://esm.sh/unist-lsp@?0"
```

Comment:
```suggestion
import * as unistLsp from "https://esm.sh/unist-lsp@?0"
```

As there are so many exports, which are documented below, I believe it is better to not overwhelm users with all the possible options

Reply:
Even with the shorter names? I generally prefer named imports over wildcard imports.

Reply:
For like 2 it is fine. Maybe 3? But 4 or more feels like a bit much to me.

Not very important to me though.